haskell/nettest/Nettest.hs

```diff
@@ -344,3 +345,20 @@ scNettestScenario constructInitialStorage stablecoinContract originateTransferli
   burnScenario
   permissionReassigmentScenario
   transferlistScenario
+
+callFrom'
```

Comment:
I'd add a comment here explaining why this function is necessary and saying that it may become unnecessary because we may reconsider upstream `callFrom`.

Reply:
Done